andela/tesla-ah
# #166240996 - CRUD articles  With API Documentation and Tests

#### What does this PR do?
In this PR, I implemented the endpoints that allow a user to be able to create, to get all articles
to get one article based on a slug
#### Description of Task to be completed?
Having the following endpoints working:
`POST /api/articles`
`GET /api/articles`
`GET /api/articles/:slug`
`PUT /api/articles/:slug`
`DELETE /api/articles/:slug`
#### How should this be manually tested?
- clone the repository
- go to your terminal and then write the following command
  `cd tesla-ah`
- after that try to install all dependencies that are required for this project, simply by running the following command:
   `yarn install`  or `yarn`
- Then running the server, by running the below commands:
   1. `sequelize db:migrate` for this command make sure that the `sequelize-cli` package is installed globally
   2. `yarn dev`
- for running the tests, write the below command in your terminal
     `yarn test`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166240996](https://www.pivotaltracker.com/n/projects/2350798/stories/166240996)
#### Screenshots (if appropriate)
N/A
#### Questions:
